src/platform_impl/linux/x11/window.rs

```diff
@@ -1407,7 +1410,37 @@ impl UnownedWindow {
         Err(ExternalError::NotSupported(NotSupportedError::new()))
     }
 
+    /// Moves the window while it is being dragged.
     pub fn drag_window(&self) -> Result<(), ExternalError> {
+        self.drag_initiate(8) // _NET_WM_MOVERESIZE_MOVE
```

Comment:
Could you define this constant with the others below? Could put them into `util.rs`. Could put them into `util/wm.rs`.